js/core/index.d.ts

```diff
@@ -22,3 +22,6 @@ export type Xor<T1, T2 = never, T3 = never, T4 = never, T5 = never, T6 = never,
 export type DeepPartial<T> = {
   [P in keyof T]?: DeepPartial<T[P]>;
 };
+
+// Omit does not exist in TS < 3.5.1
+export type Except<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>;
```

Comment:
```suggestion
export type Skip<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>;
```
It seems to be more consistent as `Pick`, `Exclude`, `Omit` are verbs too